install.sh

```diff
@@ -1,25 +1,30 @@
 # !/bin/bash
 
+# choose type
+echo "[1] Back-End"
+echo "[2] Front-End"
+read -p "choose your install type[1|2 or skip install]: " type
+
 # install brew
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+eval "$(/opt/homebrew/bin/brew shellenv)"
 
-# install via brew
-brew bundle --file=./Brewfile
+brew bundle --file=./common.BrewFile
 
-sudo xattr -dr com.apple.quarantine /Applications/GitKraken.app
-sudo xattr -dr com.apple.quarantine /Applications/MySQLWorkbench.app
-sudo xattr -dr com.apple.quarantine /Applications/Postman.app
-sudo xattr -dr com.apple.quarantine /Applications/Google\ Chrome.app
-sudo xattr -dr com.apple.quarantine /Applications/iTerm.app
-sudo xattr -dr com.apple.quarantine /Applications/KakaoTalk.app
-sudo xattr -dr com.apple.quarantine /Applications/Owly.app
-sudo xattr -dr com.apple.quarantine /Applications/IntelliJ\ IDEA.app
-sudo xattr -dr com.apple.quarantine /Applications/Slack.app
-sudo xattr -dr com.apple.quarantine /Applications/Docker.app
+if [ "$type" = "1" ]; then
+	brew bundle --file=./be.BrewFile
+elif [ "$type" = "2" ]; then
+	brew bundle --file=./fe.BrewFile
+fi
 
 # configure aws-cli
+eval "$(/opt/homebrew/bin/aws shellenv)"
 aws configure
 
 # configure zsh
 chmod 755 ./zsh/install.sh
+eval "$(/opt/homebrew/bin/zsh shellenv)"
 ./zsh/install.sh
+chsh -s /bin/zsh
+
+printf '\n\n🎉 Congrat! Your mac has been set up successfully for working with B.S!\n'
```

Comment:
파일의 맨 끝에는 개행을 넣어주세요:)
[참고](https://velog.io/@doondoony/posix-eol)
```suggestion
printf '\n\n🎉 Congrat! Your mac has been set up successfully for working with B.S!\n'

```